app/controllers/users_controller.rb

```diff
@@ -1,5 +1,10 @@
 class UsersController < ApplicationController
-  def index; end
+  def index
+    @users = User.all
+  end
 
-  def show; end
+  def show
+    @user = [User.find(params[:id])]
+    @posts = @users[0].recent_posts
+  end
```

Comment:
- [ ] Great job so far 👏. However, when I click on a user to see the user show page, I get this error: `undefined method `[]' for nil:NilClass`. At line `7` you assigned the queried user with the instance variable `@user` but at line `8` instead of `@user` you have `@users` with a `s`. Please ensure to fix that. :ok_hand: 
### error :point_down: 
![Screenshot from 2023-10-13 16-05-51](https://github.com/supreeti/BlogApp-ROR/assets/89228743/880df5ee-5b65-4e2b-9001-915bef596304)



- [ ] And after fixing the instance variable there are other errors like `undefined method photo`.  Please make sure to fix all the errors and that everything is working and looking as expected by following the wireframes. 😊 :ok_hand: 
### error :point_down: 
![Screenshot from 2023-10-13 16-38-49](https://github.com/supreeti/BlogApp-ROR/assets/89228743/14d4c196-94d5-431d-9324-848945ad667c)